tests/wamr-test-suites/test_wamr.sh

```diff
@@ -469,6 +469,37 @@ function wasi_test()
     echo "Finish wasi tests"
 }
 
+function wasi_test()
+{
+    echo  "Now start wasi tests"
+    
+    cd ${WORK_DIR}
+    if [ ! -d "wasi-testsuite" ]; then
+        echo "wasi not exist, clone it from github"
+        git clone -b prod/testsuite-base \
+            --single-branch https://github.com/WebAssembly/wasi-testsuite.git
+    else
+        echo "wasi exists, pulling updates from github"
+        pushd wasi-testsuite
+        git fetch
+        git reset --hard origin/prod/testsuite-base
+        popd
+    fi
+
+    cd wasi-testsuite
+    python3 -m pip install -r test-runner/requirements.txt
+    IWASM_PATH=$(dirname ${IWASM_CMD})
+    PATH=${PATH}:${IWASM_PATH} python3 test-runner/wasi_test_runner.py \
+                -r adapters/wasm-micro-runtime.sh \
+                -t \
+                    tests/c/testsuite/ \
+                    tests/assemblyscript/testsuite/ \
+                | tee -a ${REPORT_DIR}/wasi_test_report.txt
+    [[ ${PIPESTATUS[0]} -ne 0 ]] && exit 1
```

Comment:
does it mean that in case of failure the L500 won't be executed? Is that desired? If we don't have the last line, we probably wouldn't know from this log file whether the test execution completed or just broke half way through.

Reply:
I was replicating what's done already for spec tests https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/tests/wamr-test-suites/test_wamr.sh#L450. I'll update it in both cases to add something to the log file before exiting.